prusa_slicer_post_processing_script.py

```diff
@@ -430,27 +437,34 @@ def getPtfromCmd(line:str)->Point:
     if (x is not None) and (y is not None):
         p=Point(x,y)
     else:
-        p=None    
-    return p        
+        p=None
+    return p
 
 def makePolygonFromGCode(lines:list)->Polygon:
     pts=[]
+    wiping=False
     for line in lines:
-        if ";WIPE" in line:
-            break
-        if "G1" in line:
+        if line.startswith(";WIPE_END"):
+            wiping=False
+        elif line.startswith(";WIPE_START"):
+            wiping=True
+
+        if wiping:
+            continue
```

Comment:
Continues searching for points after the wipe, rather than breaking when encountering one.